### ISSUE TITLE
DPL-434 - As a user I need a new column in the sample sheet called Demultiplex Barcodes as this is required for SMRT Link 11

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-service#784]
**User story**
As a user I need a new column in the sample sheet called Demultiplex Barcodes as this is required for SMRT Link 11

As a user I would like a new 'Demultiplex Barcodes' column to replace the 'Generate HiFi Reads' column.

This column should be set to 'Do not Generate' if `sample_is_barcoded` is false (Ie. the sample is untagged or has one of the 'secret' tagsets) but the value of 'Generate HiFi Reads' otherwise.

I think this is as simple as:
- Rename the column in the v11 config
- Add validation, possibly dependent on run version config options.
- Maybe: alias the field in the resource

Orignal request:
> 
> Bug: Imported barcoded libraries not recognised as barcoded.
> New header required in csv file Demultiplex Barcodes
> If ‘Sample is barcoded’ = TRUE then Demultiplex Barcodes = On Instrument
> Else Do Not Generate
> This is almost replacing the Generate HiFi Reads option in #1 within the Run Setup (per well) in Traction. Depending on how easy this is to do, it would be nice to repurpose the Generate HiFi Reads option to Demultiplex barcodes. This would be auto populated based on the ‘Sample is barcoded’ flag but with the option to switch between the possible three options (On Instrument, In SMRT Link, Do Not Generate). If library/pool not barcoded or from one of the ‘restricted’ Tag groups (Nextera/ truseq/ Isoseq) then field should be locked to Do Not Generate.

**Who are the primary contacts for this story**
Craig / James

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] Add field demultiplex barcodes to pacbio well in json column smrt_link_options
- [x] Should be one of ['In SMRT Link', 'On Instrument', 'Do Not Generate']
- [x] demultiplex barcodes should appear on sample sheet

NB dependent filling is not in scope for first release.

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
